packages/tags/src/Tags.ts

```diff
@@ -55,6 +56,15 @@ export class Tags extends FocusVisiblePolyfillMixin(SpectrumElement) {
 
     constructor() {
         super();
+        new MutationController(this, {
+            config: {
+                childList: true,
+                subtree: true,
+            },
+            callback: () => {
+                this.rovingTabindexController.changeDefaultItemFocus();
+            },
+        });
```

Comment:
This is definitely the direction that I'm thinking!

One step further, if this was a part of `FocusGroup` would it "just work" for every pattern leveraging this, e.g. the Action Group listed in the related issue?